pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -1288,6 +1288,25 @@ func (o *OCMProvider) AddProperty(cluster *spi.Cluster, tag string, value string
 	return nil
 }
 
+// Get a specific cluster property
+func (o *OCMProvider) GetProperty(clusterID string, property string) (*string, error) {
+	clusterClient := o.conn.ClustersMgmt().V1().Clusters().Cluster(clusterID)
+	response, err := clusterClient.Get().Send()
+	if err != nil {
+		return nil, err
+	}
+
+	cluster := response.Body()
+	properties, _ := cluster.GetProperties()
+	propertyValue, exist := properties[property]
+
+	if !exist {
+		return nil, fmt.Errorf(fmt.Sprintf("cluster property: %s is undefined", property))
```

Comment:
```suggestion
		return nil, fmt.Errorf("cluster property: %s is undefined", property)
```
errorf acts as sprintf but creates an error instead of a string

Reply:
Resolved in https://github.com/openshift/osde2e/pull/1558/commits/0de34542c4d705ab0513096a30d520fbf86d25bb